hbase-assembly/pom.xml

```diff
@@ -76,7 +76,7 @@
           <finalName>hbase-${project.version}</finalName>
           <skipAssembly>false</skipAssembly>
           <appendAssemblyId>true</appendAssemblyId>
-          <tarLongFileMode>gnu</tarLongFileMode>
+          <tarLongFileMode>posix</tarLongFileMode>
```

Comment:
Have any requirement about the mvn version ? If not too new mvn version, I think it's OK (+1). (Sorry, I'm not quite familiar with this option.